File: app/src/main/java/com/prideven/android/HungryEats/LaunchActivity.java
package com.prideven.android.hungryeats;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class LaunchActivity extends AppCompatActivity {
    protected void onCreate(@Nullable Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button orderButton = findViewById(R.id.btnOrder);

        orderButton.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                startActivity(new Intent(v.getContext(), LoginAndRegisterActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/prideven/android/HungryEats/LaunchActivity.kt
package com.prideven.android.hungryeats

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class LaunchActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val orderButton = findViewById<Button>(R.id.btnOrder)
        orderButton.setOnClickListener { v ->
            startActivity(
                Intent(
                    v.context,
                    LoginAndRegisterActivity::class.java
                )
            )
        }
    }
}